### PR TITLE
Add to the rm category

### PR DESCRIPTION
**Pull request by vitorhcl**

### CODE REVIEW

**Comment by ccoVeille** [comment]
May I recommend also

https://github.com/theimpossibleastronaut/rmw

I'm using it

File: data/apps.csv [hunk 1]
```diff
@@ -1430,7 +1430,6 @@ terminal,extraterm,https://extraterm.org/,https://github.com/sedwards2009/extrat
 terminal,st,https://st.suckless.org/,https://git.suckless.org/st/,A simple terminal implementation for X.
 terminal,wterm,,https://github.com/TornadoCookie/wterm,A native Wayland terminal emulator based on an st fork using wld.
 terminal,mlterm,https://mlterm.sourceforge.net/,https://github.com/arakiken/mlterm,"A very fast low latency terminal emulator with features such as rendering variable width fonts, proper bidirectional support out of the box, a daemon mode, multiple XIM, and true background transparency."
-games,sshattrick,,https://github.com/ricott1/sshattrick,Play Hattrick in your terminal over SSH.
```
Comment (ccoVeille) [inline data/apps.csv:1433]: I'm unsure why this one was removed. A conflict issue? Or maybe I'm missing something

**Review by ccoVeille — CHANGES_REQUESTED** [review]

File: data/apps.csv (hunk 1 above)
Comment (vitorhcl) [reply data/apps.csv:1433]: Yeah, it was by accident

**Comment by vitorhcl** [comment]
> May I recommend also
> 
> https://github.com/theimpossibleastronaut/rmw
> 
> I'm using it

Could you submit a suggestion please?

**Comment by ccoVeille** [comment]
> > May I recommend also
> > 
> > https://github.com/theimpossibleastronaut/rmw
> > 
> > I'm using it
> 
> Could you submit a suggestion please?

I'll no worry.

**Comment by toolleeo** [comment]
Thanks for the contribution!

**Comment by toolleeo** [comment]
> > > May I recommend also
> > > https://github.com/theimpossibleastronaut/rmw
> > > I'm using it
> > 
> > 
> > Could you submit a suggestion please?
> 
> I'll no worry. I saw your PR and noticed rmw was not in it. I'll open a PR once yours will be merged. No need to wait or add confusion

Thanks for your patience.
I have merged the changes.
You can proceed with a clean pull request anytime.

**Comment by vitorhcl** [comment]
> May I recommend also
> 
> https://github.com/theimpossibleastronaut/rmw
> 
> I'm using it

@ccoVeille I've just noticed the tool is already on the repo 😀

**Comment by ccoVeille** [comment]
> > May I recommend also
> > 
> > https://github.com/theimpossibleastronaut/rmw
> > 
> > I'm using it
> 
> @ccoVeille I've just noticed the tool is already on the repo 😀

Thanks then. Did you set it in the right category then?

**Comment by ccoVeille** [comment]
It's OK. All good then

**Pull request merged**